backend/src/main/java/com/hackathon/backend/service/FavoriteProductService.java

```diff
@@ -0,0 +1,21 @@
+package com.hackathon.backend.service;
+
+import com.hackathon.backend.model.FavoriteProducts;
+import com.hackathon.backend.repository.FavoriteProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class FavoriteProductService {
+
+    private final FavoriteProductRepository favoriteProductRepository;
+
+    @Autowired
+    public FavoriteProductService(FavoriteProductRepository favoriteProductRepository) {
+        this.favoriteProductRepository = favoriteProductRepository;
+    }
+
+    public FavoriteProducts insertFavoriteProducts(FavoriteProducts favoriteProducts) {
```

Comment:
We used to get a dto at the controller level and pass it to the service level, where that dto has to be converted into a model entity. So can you create FavoriteProductsDto, pass it to FavoriteProductService, convert it to FavoriteProducts and store it in favoriteProductRepository?